### PR TITLE
[PERF] core: Blazing Fast

### PR DESCRIPTION
**Pull request by Julien00859**
Gotta go fast!

### CODE REVIEW

**Comment by Julien00859** [comment]
Too sad :(

**Comment by amigrave** [comment]
@VincentSchippefilt I tested the @Julien00859 's branch and the numbers logged for each request are just incredible ! Why closing the PR of the best performance patch to this date ?

**Review by sdegueldre — COMMENTED** [review]
28 days too late for this one unfortunately.

**Comment by sdegueldre** [comment]
As a side not I propose that we remove the line entirely and use common sense instead of numbers when optimizing code, I have it on good authority that this makes the most sense both technically and from a business standpoint.

**Comment by william-andre** [comment]
> 28 days too late for this one unfortunately.

337 days early*

**Comment by william-andre** [comment]
I am a bit surprised by the time it takes to run the tests. Even though the number of queries went down by a lot, the time is almost still the same.
Maybe we should go for a full Assembly approach instead of using Python?

File: odoo/netsvc.py [hunk 1]
```diff
@@ -78,7 +78,7 @@ def filter(self, record):
             query_time = threading.current_thread().query_time
             perf_t0 = threading.current_thread().perf_t0
             remaining_time = time.time() - perf_t0 - query_time
-            record.perf_info = '%s %s %s' % self.format_perf(query_count, query_time, remaining_time)
+            record.perf_info = '%s %s %s' % self.format_perf(0, 0, 0)
```
Comment (william-andre) [inline odoo/netsvc.py:81]:
```suggestion
            record.perf_info = '%s %s %s' % self.format_perf(
                query_count=0,
                query_time=0,
                remaining_time=0,
            )
```

**Comment by VincentSchippefilt** [comment]
sorry, but I have it on good authority that less code is better. I can't consider this PR until there is a net negative amount of lines of code... May be remove 77 to 80 ?

**Comment by ged-odoo** [comment]
remember that we have a target of -30% lines of code. With JS, it is easy because we can just concatenate any line of code, but it is more difficult in python. You can start by removing comments and white lines. @VincentSchippefilt

**Pull request closed without merging**